src/main/resources/liquibase/workbench_changelog/12_5_0.xml

```diff
@@ -0,0 +1,713 @@
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (6,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (11,3,0);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (13,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (14,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (15,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (16,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (17,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (18,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (19,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (20,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (21,3,1);
+			INSERT INTO role_type_permission (permission_id, role_type_id, selectable) VALUES (22,3,1);
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-21">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				select count(1)
+				from role_permission
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			INSERT INTO table role_permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO role_permission (permission_id, role_id) VALUES (1,1);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (1,5);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (10,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (4,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (5,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (6,2);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (10,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (4,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (5,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (6,3);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (14,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (17,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (18,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (19,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (20,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (21,4);
+			INSERT INTO role_permission (permission_id, role_id) VALUES (22,4);
+		</sql>
+	</changeSet>
+
+
+	<changeSet author="vanina" id="v12.5.0-22">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				select count(1) from role where id = 5;
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Update table role where row is SUPERADMIN
```

Comment:
Update the comment too be less generic like: make SUPERADMIN non-editable and non-assignable